content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,837 @@
+---
+linkTitle: Vertex Deformation for Vegetation Bending Tutorial
+title: Vertex Deformation for Vegetation Bending Tutorial
+description: A tutorial for add vegetation bending with the Atom rendered in the Open 3D Engine (O3DE).
+toc: true
+---
+
+In this tutorial, we cover how to make your own material type and how to edit vertex shaders to achieve a vegetation bending effect. While we use vegetation bending as an example, the primary goal of this tutorial is to familiarize yourself with how to use and create custom material types and vertex shaders. 
+
+This tutorial covers the following concepts:
+* Making your own material type
+* Creating materials in the **Material Editor**
+* Editing vertex shaders
+* Adding editable properties for your material type
+* Using optional vertex streams
+* Adding passes to your material type
+* Using the **Pass Tree Visualizer** to debug passes
```

Comment:
Reorganized the concepts so we can see how each is connected better

```suggestion
This tutorial covers the following concepts:
* Making your own material type
* Creating materials in the **Material Editor**
* Editing vertex shaders
* Adding editable properties for your material type
* Using optional vertex streams
* Adding passes to your material type
* Using the **Pass Tree Visualizer** to debug passes
```

Reply:
I understand what you mean about "using shader options," so I updated the docs to put a little more emphasis on "vertex streams." I kept the concept as "Using optional vertex streams" and renamed the section to "Use an optional vertex stream" because the shader option that I was adding was only a part of adding the vertex stream -- the vertex color is part of the vertex stream and I only use the shader option from whether the vertex stream is available, so it seems more appropriate to use "optional vertex stream." 

Otherwise, I took your suggestions for italicizing and indenting some concepts!